src/Microsoft.Identity.Client/Platforms/net45/PlatformParameters.cs

```diff
@@ -50,7 +50,5 @@ public PlatformParameters(object ownerWindow)
         /// Gets the owner of the browser dialog which pops up for receiving user credentials. It can be null.
         /// </summary>
         public object OwnerWindow { get; private set; }
-
-        internal bool UseHiddenBrowser { get; set; }
```

Comment:
You've removed _UseHiddenBrower_ from Net45:_PlatformParameters_, but added a _UseHiddenBrowser_ property to WinRT:_PlatformParameters_. Why are they different?